presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -256,6 +256,17 @@ class SystemConfig : public ConfigBase {
   /// this config only applies if the memory arbitration has been enabled.
   static constexpr std::string_view kQueryMemoryGb{"query-memory-gb"};
 
+  /// Specifies the amount of query memory capacity reserved to ensure that a
```

Comment:
How would you typically determine this value ? Whats a good back of the envelope calculation for this ?

Reply:
It can be based on the number of concurrent running queries * reserved memory pool capacity.